lib/node_modules/@stdlib/stats/base/dists/arcsine/mode/include/stdlib/stats/base/dists/arcsine/mode.h

```diff
@@ -0,0 +1,42 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#ifndef STDLIB_STATS_BASE_DISTS_ARCSINE_MODE_H
+#define STDLIB_STATS_BASE_DISTS_ARCSINE_MODE_H
+
+/*
+* If C++, prevent name mangling so that the compiler emits a binary file having undecorated names, thus mirroring the behavior of a C compiler.
+*/
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+* Returns the mode of an arcsine distribution with minimum support `a` and maximum support `b`.
+*
+* @param a    minimum support
+* @param b    maximum support
+* @return     mode of the distribution
+*/
```

Comment:
We don't include parameter and return descriptions in header files to keep them concise.

```suggestion
/**
* Returns the mode of an arcsine distribution with minimum support `a` and maximum support `b`.
*/
```